ci/base-images/debian/Dockerfile.dotnet8

```diff
@@ -17,6 +17,7 @@ ENV PATH=${PATH}:/root/.nvm/versions/node/v${NODE_VERSION}/bin:${JAVA_HOME}/bin:
 COPY ci/base-images/debian/install.sh /tmp/
 
 RUN apt-get update && apt-get install -qq -y --no-install-recommends curl bash bzip2 git-core zip unzip make gawk \
+        autotools-dev autoconf-archive gettext intltool libadwaita-1-dev \
```

Comment:
These are required to generate an sbom for [this](https://github.com/PintaProject/Pinta) repo and unrelated to mill.